.github/workflows/ansible-ci.yml

```
@@ -1,98 +1,71 @@
+    name: Ansible Lint
     runs-on: ubuntu-latest
     steps:
-      - name: "Checkout Repository"
+      - name: Checkout code
         uses: actions/checkout@v4
-
-      - name: "Set up Python 3.9"
-        uses: actions/setup-python@v5
+        
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)